app/src/lib/ui/Beneficiary/NotebookMembers.svelte

```diff
@@ -3,13 +3,72 @@
 		type GetNotebookByBeneficiaryIdQuery,
 		RoleEnum,
 	} from '$lib/graphql/_gen/typed-document-nodes';
-
 	import { Text } from '$lib/ui/utils';
+	import Dialog from '$lib/ui/Dialog.svelte';
+	import { accountData, token } from '$lib/stores';
+	import { postApiJson } from '$lib/utils/post';
+	import { captureException } from '$lib/utils/sentry';
+	import { trackEvent } from '$lib/tracking/matomo';
+	import { Radio } from '$lib/ui/base';
+	import { createEventDispatcher } from 'svelte';
+
+	type Notebook = GetNotebookByBeneficiaryIdQuery['notebook'][0];
+	export let members: Notebook['members'];
+	export let notebookId: Notebook['id'];
+
+	const dispatch = createEventDispatcher();
+
+	const options = [
+		{ name: 'referent', label: 'Oui' },
+		{ name: 'no_referent', label: 'Non' },
+	];
 
-	export let members: GetNotebookByBeneficiaryIdQuery['notebook'][0]['members'];
+	let memberType: 'referent' | 'no_referent' = 'no_referent';
+
+	async function addCurrentAccountToNotebookMembers() {
+		trackEvent('notebook membership', 'join notebook members');
```

Comment:
on pourrait conserver un peu de cohérence avec les anciens event
```suggestion
		trackEvent('pro',  'members', 'join_notebook_members');
```